ui/app/AppLayouts/Communities/panels/MembersTabPanel.qml

```diff
@@ -81,21 +89,23 @@ Item {
                 ]
 
                 filters: AnyOf {
```

Comment:
I changed the search logic here a bit, so that:
- we match a substring of nickname/displayName (in the same order that we display them in the delegates)
- we match the full compressed pub key (and not just the substring) because with many users, this might lead to many false positives
- do not match against the regular (non compressed) pubkey as we don't display that info anywhere, and from various places, users can only copy others' compressed pubkey

Is that ok @jrainville @benjthayer @xAlisher ?

Reply:
Yeah good idea; pls create the issue, feel free to assign to me